src/repository/storage/postgres/orm/sequelize/oauthProviders.ts

```diff
@@ -0,0 +1,67 @@
+import { Model, DataTypes, Sequelize, InferAttributes, InferCreationAttributes, CreationOptional } from 'sequelize';
+import Orm from '@repository/storage/postgres/orm/sequelize/index.js';
+
+/**
+ * Class representing oauth providers model in database
+ */
+export class OAuthProvidersModel extends Model<InferAttributes<OAuthProvidersModel>, InferCreationAttributes<OAuthProvidersModel>> {
+  /**
+   * Provider id
+   */
+  public declare id: CreationOptional<number>;
+
+  /**
+   * Provider name
+   */
+  public declare name: string;
```

Comment:
descriptions are not clear. What will be stored here?

Reply:
Changed